themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties

```diff
@@ -310,6 +337,14 @@ web-origins=Web オリジン
 web-origins.tooltip=許可される CORS オリジンを設定します。有効なリダイレクト URI のすべてのオリジンを許可するには 「+」 を追加してください。すべてのオリジンを許可するには 「*」 を追加してください。
 fine-oidc-endpoint-conf=OpenID Connect の詳細設定
 fine-oidc-endpoint-conf.tooltip=このセクションを展開して、OpenID Connect プロトコルに関連するこのクライアントの高度な設定を行います。
+access-token-signed-response-alg=アクセストークン署名アルゴリズム
+access-token-signed-response-alg.tooltip=アクセストークンの署名に使用されるJWAアルゴリズム。
+id-token-signed-response-alg=IDトークン署名アルゴリズム
+id-token-signed-response-alg.tooltip=IDトークンの署名に使用されるJWAアルゴリズム。
+id-token-encrypted-response-alg=IDトークン暗号化鍵管理アルゴリズム
+id-token-encrypted-response-alg.tooltip=IDトークンの暗号化鍵の管理に使用されるJWAアルゴリズム。このオプションは、暗号化されたIDトークンが必要な場合にのみ必須です。空のままにすると、IDトークンは署名されますが、暗号化されません。
+id-token-encrypted-response-enc=IDトークン暗号化コンテンツの暗号化アルゴリズム
+id-token-encrypted-response-enc.tooltip=id-token-encrypted-response-enc.tooltip=IDトークンの暗号化の際に、コンテンツの暗号化に使用されるJWAアルゴリズム。このオプションは、暗号化されたIDトークンが必要な場合にのみ必須です。空のままにすると、IDトークンは署名されますが、暗号化されません。
```

Comment:
@k-tamura Remove `id-token-encrypted-response-enc.tooltip=` ? It looks like original English message has same issue.
https://github.com/keycloak/keycloak/blob/master/themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties#L349

Reply:
Thanks. I will fix it too: https://github.com/keycloak/keycloak/pull/6480